catalog/spatial/csw/spatial-csw-source-common/src/test/java/org/codice/ddf/spatial/ogc/csw/catalog/common/source/reader/TestGetRecordsMessageBodyReader.java

```diff
@@ -95,7 +91,7 @@ public void testConfigurationArguments() throws Exception {
         reader.readFrom(CswRecordCollection.class, null, null, null, httpHeaders, is);
 
         // Verify the context arguments were set correctly
-        verify(mockProvider, times(3)).unmarshal(any(HierarchicalStreamReader.class),
+        verify(mockProvider, times(1)).unmarshal(any(HierarchicalStreamReader.class),
```

Comment:
Before it was using an actual transformer which was not necessary for this unit test. By not using the actual transformer i was able to untangle some of the dependency mess. Instead I just used a mock to return one result